adding_species_manuscript.md

```diff
@@ -389,31 +391,238 @@ Table of species/demographic scenarios added/worked on by community since origin
+* The ```species.py``` is the file that we will need to edit with our species specific information. The basics that we will need here include:
+  * 1) *chromosome-specific* recombination rates (could also be single rate)
+  * 2) a genome-wide average *mutation rate*
+  * 3) a *generation time* estimate (in years)  
+  * 4) a default *population size*
+  * 5) literature citations that one can point to for the above and the assembly
+  
+- Test the code while building the files
+
+```stdpopsim``` in part guarantees code quality through the use of unit testing. Basic sanity tests for this new species will be completed through QC over in the ```tests/test_AnoGam.py``` file created by the command ```python -m maintenance add-species Anopheles_gambiae``` above.
+
+To run the tests in stdpopsim uses the pytest module. We will quickly run the tests that our maintenance function has stubbed out and expect it to error out:
+
+```
+python -m pytest tests/test_AnoGam.py
+```
+The test will tell that the required information entered into this species definition is missing.
+Let's add it.
+
+### 1) Recombination rates
+As a reference for recombination rates in *Anopheles gambiae* we use a recombination map based on a study from Pombi et al. (2006). In that manuscript the authors cite rates around 1cM/Mb, with a bit of variation among arms. In particular let's edit the following block of code in species.py that defines the recombination_rate dict:
+
+```
+_recombination_rate = {
+    "2L": 0, #setting to zero because of inversion 
+    "2R": 1.3e-8, 
+    "3L": 1.6e-8, 
+    "3R": 1.3e-8, 
+    "X": 1e-8,
+    "Mt": 0
+}
+```
+Every information in regards to the population genetics parameters requires a citeable reference. To do that we can create a stdpopsim.Citation object to the same species.py file. that Citation object looks like this:
+
+```
+_PombiEtAl = stdpopsim.Citation(
+    doi="https://doi.org/10.4269/ajtmh.2006.75.901",
+    year=2006,
+    author="Pombi et al.",
+    reasons={stdpopsim.CiteReason.REC_RATE},
+)
+```
+
+To verify if we have progressed on the tests units, we can type again:
+
+```
+python -m pytest tests/test_AnoGam.py
+```
+### 2) Mutation rates
+
+Editing the species.py file concerning the mutation rate is very similar to the recombination rate. It turns out that there is no great estimates for mutation rate in Anopheles but in a recent population genomics effort by the Ag1000G consortium it has relied on Drosophila estimates from Schrider et al. (2013) that set u=5.49e-9. We'll go with that:
+
+```
+_overall_rate = 5.49e-9
+_mutation_rate = {
+    "2L": _overall_rate,
+    "2R": _overall_rate,
+    "3L": _overall_rate,
+    "3R": _overall_rate,
+    "X": _overall_rate,
+    "Mt": _overall_rate
+}
+```
+We should again add a citation for the mutation rate and then edit the _mutation_rate dict that was templated for us:
```

Comment:
"and then edit the _mutation_rate dict that was templated for us" redundant with above?

Reply:
done